broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java

```diff
@@ -1636,4 +1649,14 @@ private MessageExtBrokerInner toMessageExtBrokerInner(MessageExt msgExt) {
         inner.setWaitStoreMsgOK(false);
         return inner;
     }
+
+    private boolean validateSlave(RemotingCommand response) {
```

Comment:
I think the method name `validateSlave` meaning only check but not modify, maybe we should use a name mean `adjust response if slave`